ingestion/src/metadata/ingestion/api/parser.py

```diff
@@ -329,7 +329,10 @@ def parse_workflow_config_gracefully(
         workflow_config = OpenMetadataWorkflowConfig.parse_obj(config_dict)
         return workflow_config
 
-    except ValidationError:
+    except ValidationError as e:
+        if e.model.__name__ == OpenMetadataWorkflowConfig.__name__:
```

Comment:
Hi, I think I understand what we're trying to do here. The issue as that any misconfig we pass, for example, inside the `serviceConnection.config`, will still be raised by the model name `OpenMetadataWorkflowConfig`, so we would be losing the manual parsing we do below.

This solution shadows the later behavior. I think we need to find another approach